cypress/integration/regressionTs/setEntityActions.spec.ts

```diff
@@ -1,12 +1,12 @@
-import * as util from '../support/utilities'
-import constants from '../support/constants'
-import s from '../support/selectors'
+import * as util from '../../support/utilities'
+import constants from '../../support/constants'
+import s from '../../support/selectors'
 
 describe('Set Entity Actions', () => {
     const testData = {
         modelName: `setEntity`,
         modelFile: 'setEntity.cl',
-        enumValueCount: 14,
+        enumValueCount: 11, // 14,
```

Comment:
I'm not sure why 14 worked previously and fails now, but 11 seems correct. Maybe we fixed a bug and this was dependent on that bug.